### ISSUE TITLE
[dcl.init] should this 'declarator' be the 'initializer'8

### ISSUE THREAD

**Issue by Heuristack** [cplusplus/draft#1615]
Hi, I got confused when reading [dcl.init]
> A declarator can specify an initial value for the identifier being declared.

Should it be like

An **initializer** can specify an initial value for the identifier being declared.

Noticed that the BNF reflecting the relationship between a **declarator** and an **initializer**, 
```
\begin{bnf}
\nontermdef{init-declarator}\br
    declarator initializer\opt
\end{bnf}
```

FYI, @sdutoit, thanks!

**Comment by tkoeppe** [comment]
I suppose a "declarator" (English) isn't the same as a "*declarator*" (grammar production).

**Comment by jensmaurer** [comment]
@tkoeppe, I believe you're confused here.

**Comment by jensmaurer** [comment]
[dcl.decl] p1 clearly distinguishes "declarator" and "initializer", and I fail to see a use of an English-level "declarator" anywhere. (Some of these words are not \grammarterm'ed, but probably should be.)

The note in [dcl.decl] p3 muddies the waters a bit, though.

Back to the original issue: I think this should be "An _init-declarator_ can specify an initial value ..." In this context, an initializer always specifies an initial value, so it doesn't fit the "can" in the sentence.

**Comment by Heuristack** [comment]
I don't think #1616 is a proper fix for this issue and proposed #1617.

> In this context, an initializer always specifies an initial value, so it doesn't fit the "can" in the sentence.

I think "can" here means an initializer is capable of specifying an initial value, and it "can" to used to do that or not, since it is optional (for a declarator).
